packages/autocomplete-core/src/onInput.ts

```diff
@@ -29,6 +29,11 @@ interface OnInputParams<TItem extends BaseItem>
 
 const runConcurrentSafePromise = createConcurrentSafePromise();
 
+let nextStateRef: Partial<AutocompleteState<any>> = {};
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Casting is necessary because `{}` doesn't fulfil the `InternalAutocompleteOptions` contract. We don't care since we'll reassign it as soon as `onInput` executes, we don't need defaults.